shared/js/ui/models/site-company-list.es6.js

```diff
@@ -17,18 +18,14 @@ SiteCompanyList.prototype = window.$.extend({},
 
         fetchAsyncData: function () {
             return new Promise((resolve, reject) => {
-                this.sendMessage('getCurrentTab').then((tab) => {
-                    if (tab) {
```

Comment:
Is this check not necessary any more?

Reply:
er maybe for _updateCompaniesList let me double check. Otherwise it's functionally equiv.

Reply:
For paranoia sake, I added it back in. Pretty sure it's not needed.